### PR TITLE
Add publish test of PyPi cu100mkl

### PR DESCRIPTION
**Pull request by stu1130**
## Description ##
Add PyPi cu100mkl CI test  to avoid manual test when updating the dependencies

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
1. add PyPi cu100mkl test
2. align scala naming convention with PyPi

## Comments ##
@lanking520  @szha

### CODE REVIEW

**Comment by piyushghai** [comment]
Thanks for your contributions @stu1130. 
@mxnet-label-bot Add [CI, Test]

File: ci/jenkins/Jenkinsfile_unix_gpu [hunk 1]
```diff
@@ -57,8 +57,9 @@ core_logic: {
     custom_steps.test_unix_cpp_mkldnn_gpu(),
     custom_steps.test_unix_python3_integration_gpu(),
     custom_steps.test_unix_cpp_package_gpu(),
-    custom_steps.test_unix_scala_gpu(),
-    custom_steps.test_unix_distributed_kvstore_gpu()
+    custom_steps.test_unix_distributed_kvstore_gpu(),
+    custom_steps.test_static_python_gpu()
```
Comment (lanking520) [inline ci/jenkins/Jenkinsfile_unix_gpu]: Why remove scala_gpu test?

Comment (stu1130) [reply ci/jenkins/Jenkinsfile_unix_gpu]: delete it by accident add it back

**Comment by lanking520** [comment]
Please try to collect all your commit and push it once, the CI consumption for that is very expensive.

**Comment by Roshrini** [comment]
@lanking520 Can you take another look at this PR?
@marcoabreu @zachgk Can you help review?

**Comment by wkcn** [comment]
Merged. Thank you!

**Pull request merged**